public/app/plugins/datasource/phlare/dataquery.gen.ts

```diff
@@ -16,11 +16,24 @@ export type PhlareQueryType = ('metrics' | 'profile' | 'both');
 
 export const defaultPhlareQueryType: PhlareQueryType = 'both';
 
-export interface GrafanaPyroscope extends common.DataQuery {
+export interface GrafanaPyroscope {
+  /**
+   * For mixed data sources the selected datasource is on the query level.
+   * For non mixed scenarios this is undefined.
+   * TODO find a better way to do this ^ that's friendly to schema
+   * TODO this shouldn't be unknown but DataSourceRef | null
+   */
+  datasource?: unknown;
```

Comment:
cuetsy is no longer noticing that `common.DataQuery` should be expressed via `extends` statement. Instead, it's inlining all the fields directly into the generated types. This requires a fix in cuetsy.

I'm fairly sure this is happening because of changes in [the CUE value actually being passed to cuetsy](https://github.com/grafana/thema/blob/main/lineage.cue#L258) - passing a reference to a field (`schema`) that's unified with a user-provided struct literal. Will need digging to figure it out, though.